staging/src/k8s.io/apimachinery/pkg/util/duration/duration.go

```diff
@@ -57,17 +57,29 @@ func HumanDuration(d time.Duration) string {
 	}
 	minutes := int(d / time.Minute)
 	if minutes < 10 {
-		return fmt.Sprintf("%dm%ds", minutes, int(d/time.Second)%60)
+		s := int(d/time.Second) % 60
+		if s == 0 {
```

Comment:
maybe a small test for these duration changes?

Reply:
I added new tests to the _test.go